src/scripts/main.js

```diff
@@ -1,3 +1,267 @@
+});
+
+const form = document.createElement('form');
+
+document.body.append(form);
+
+form.className = `new-employee-form`;
+
+form.insertAdjacentHTML('afterbegin', `
+  <label>Name:
+    <input
+      name="name"
+      data-qa="name"
+      type="text"
+      required
+      minlength="4"
+    >
+  </label>
+
+  <label>Position:
+    <input
+      name="position"
+      data-qa="position"
+      type="text
+      required"
+    >
+  </label>
+
+  <label>Office:
+    <select
+      name="office"
+      data-qa="office"
+    >
+      <option value="Tokyo">Tokyo</option>
+      <option value="Singapore">Singapore</option>
+      <option value="London">London</option>
+      <option value="New York">New York</option>
+      <option value="Edinburgh">Edinburgh</option>
+      <option value="San Francisco">San Francisco</option>
+    </select>
+  </label>
+  <label>Age:
+    <input
+      name="age"
+      data-qa="age"
+      type="text"
+      required
+    >
+  </label>
+
+  <label>Salary:
+    <input
+      name="salary"
+      data-qa="salary"
+      type="text"
+      required
+    >
+  </label>
+
+  <button
+    class="button"
+    type="submit"
+  >
+    Save to table
+  </button>
+`);
+
+form.addEventListener('submit', (ev) => {
+  ev.preventDefault();
+
+  const data = new FormData(form);
+
+  const salary = `$${(+data.get('salary')).toLocaleString()}`;
+  const age = +`${toNumber(data.get('age'))}`;
+  const formCheckResult = formCheck(
+    `${age}`,
+    `${data.get('name')}`,
+    `${data.get('position')}`
+  );
+
+  if (!formCheckResult) {
+    return;
+  }
+
+  tbody.insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>${data.get('name')}</td>
+      <td>${data.get('position')}</td>
+      <td>${data.get('office')}</td>
+      <td>${age}</td>
+      <td>${salary}</td>
+    </tr>
+  `);
+
+  function formCheck(ageWorker, nameWorker, positionWorler) {
+    if (nameWorker.length < 4) {
+      pushNotification(420, 350, 'Error',
+        'the name should be atlist 4 characters', 'error');
+
+      return false;
+    }
+
+    if (positionWorler.length < 4) {
+      pushNotification(420, 350, 'Error',
+        'the position should be atlist 4 characters', 'error');
+
+      return false;
+    }
+
+    if (+ageWorker < 18) {
+      pushNotification(420, 350, 'Error',
+        'the age should not be less than 18', 'error');
+
+      return false;
+    }
+
+    if (+ageWorker > 90) {
+      pushNotification(420, 350, 'Error',
+        'the age should not be more than 90', 'error');
+
+      return false;
+    }
+
+    if (+data.get('salary') < 50000) {
+      pushNotification(420, 350, 'Error!',
+        'Employee\'s salary must be greater than $50,000!', 'error');
+
+      return false;
+    }
+
+    pushNotification(420, 350, 'Succes',
+      'new employee added to the table', 'success');
+
+    return true;
+  }
+
+  form.reset();
+});
+
+function pushNotification(tops, right, title, description, type) {
```

Comment:
```suggestion
function pushNotification(tops, right, title, description, type) {
```
tops и right всегда одинаковые, может не нужно их передавать в параметрах?

Reply:
fixed